tests/python_tests/test_continuous_batching.py

```diff
@@ -82,13 +74,14 @@ def test_e2e_real_models(tmp_path, model_id):
 @pytest.mark.parametrize("prompt", batched_prompts[1:])  # num_beams=15 diverges on the first prompt.
 @pytest.mark.precommit
 def test_continuous_batching_vs_stateful(prompt, generation_config):
```

Comment:
in general, I think comparison CB vs Stateful is not required and we need to compare against HF only.

Probably, it can be done seaprately.
We need to understand whether the same test exists in test_llm_pipeline.py and either move there / drop.

Reply:
let's split to separated pr.